person.rb

```diff
@@ -1,3 +1,4 @@
+require_relative 'nameable'
 class Person
```

Comment:
Great job on implementing `require_relative 'nameable'` in this file, however, I have some suggestions for improvement:

- 🚨 You have not inherited from the `Nameable` class in your `Person` class. This means that your `Person` class does not have access to the methods and attributes of the `Nameable` class. [This violates the requirement of inheritance in Ruby](https://www.rubyguides.com/2019/01/what-is-inheritance-in-ruby/)¹². To fix this, you need to use the `<` operator to indicate that your `Person` class is a subclass of the `Nameable` class. For example, you should write:

```ruby
class Person < Nameable
  # your code here
end
```


I hope this feedback helps you improve your code and your coding skills. Please let me know if you have any questions or need further clarification. Happy coding! 😊

Reply:
Thank you for the review and feedback @topeogunleye 
I will make sure to implement the suggested changes.
Thank you 🙏🏻

Reply:
you're welcome @RileyManda. Please kindly submit for review